platform/security/sts/security-sts-guestclaimshandler/src/main/java/org/codice/ddf/security/claims/guest/config/mapping/GuestClaimsHandlerConfigMappingProvider.java

```diff
@@ -0,0 +1,167 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.security.claims.guest.config.mapping;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import org.apache.commons.io.IOUtils;
+import org.boon.Boon;
+import org.codice.ddf.config.ConfigService;
+import org.codice.ddf.config.ConfigSingleton;
+import org.codice.ddf.config.mapping.ConfigMapping;
+import org.codice.ddf.config.mapping.ConfigMappingException;
+import org.codice.ddf.config.mapping.ConfigMappingProvider;
+import org.codice.ddf.config.model.NetworkProfileConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GuestClaimsHandlerConfigMappingProvider implements ConfigMappingProvider {
+
+  private static final Logger LOGGER =
+      LoggerFactory.getLogger(GuestClaimsHandlerConfigMappingProvider.class);
+
+  private static final String PID = "ddf.security.sts.guestclaims";
```

Comment:
Do some of these constants already exists in the project? If so we should reference those

Reply:
Not for these.